src/js/EditorBlock.js

```diff
@@ -84,7 +99,47 @@ export default class EditorBlock extends Component {
             )
         }
 
-        const keyInput = (
+        const userAPIKeyMessage = this.state.isSavedKey ? (
+            <div>
+                <p style={{textAlign: 'center'}}>
+                    {__( 'Only administrators can change the Google Maps API key.')  }<br />
+                    {__( 'Note: changing the API key effects all Google Map Embed blocks.')  }
+                </p>
+                <TextControl
+                    key="api-input"
+                    value={ this.state.apiKey }
+                    onChange={ value => this.setState({ apiKey: value }) }
+                    style={{textAlign: 'center', border: 'solid 1px rgba(100,100,100,0.25)'}}
+                    readOnly={true}
```

Comment:
Wondering if we can avoid some of the code duplication here eg this could be: 

```suggestion
                    readOnly={!PantheonGoogleMapsData.userCanManageOptions}
```